docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -115,6 +115,14 @@ WildFly defined layers to assemble a custom configuration. <<galleon-cli-recipes
 
 Basic layers:
 
+* _microprofile-fault-tolerance_: Support for MicroProfile Fault Tolerance.
+* _microprofile-jwt_: Support for MicroProfile JWT.
```

Comment:
These layers are still documented in the WildFly EE layers, should be removed from EE.

Reply:
Thanks; will fix.

Reply:
Fixed.